tests/php/Control/RequestHandlingTest.php

```diff
@@ -25,16 +24,6 @@ class RequestHandlingTest extends FunctionalTest
     protected static $fixture_file = null;
 
     protected static $illegal_extensions = array(
-        // Suppress CMS error page handling
-        Controller::class => array(
-            ErrorPageControllerExtension::class,
-        ),
-        Form::class => array(
```

Comment:
Instead of removing these, we should switch to the new namespace of the new module. Is that ok @jonom ?